README.md

```diff
@@ -1,17 +1,17 @@
-# Long Form Audio Testing
-This project provides a ready-to-use template for doing in-depth testing on long-form audio.
+# **Long Form Audio Testing**
+This project provides a ready-to-use template for doing in-depth testing on audio voice apps serving long audio streams, such as music players, radio stations and games.
 
-We store the utterances that we want to test in the file `input/utterances.csv`.
```

Comment:
Add a Who Is This For section here. It should describe the use-cases of the customers that this is helpful for.

This should be one or two short paragraphs, with links to relevant additional docs and case studies.